oarepo_dashboard/ui/dashboard_communities/semantic-ui/js/dashboard_communities/search/ComputerTabletCommunitiesListItem.jsx

```diff
@@ -0,0 +1,127 @@
+  result,
+  communityTypeLabelTransparent,
+  isRestricted,
+}) => {
+  const communityType = result.ui?.type?.title_l10n;
+
+  return (
+    <Grid className="computer tablet only item result-list-item community rel-mb-1 rel-p-1">
+      <Grid.Column width={16} verticalAlign="middle" className="pl-0">
+        <div className="flex align-items-center">
+          <Image
+            wrapped
+            src={result.links.logo}
+            size="small"
+            className="community-image rel-mr-2"
+            alt=""
+          />
+          {/* TODO: where to put this style? */}
+          <div style={{ flexShrink: 100 }}>
+            {isRestricted && (
+              <div className="rel-mb-1">
+                <RestrictedLabel access={result.access.visibility} />
+              </div>
+            )}
+            {/* TODO: Need to get ui link for members in a better way */}
+            <Header as="h4">
+              <a
+                className="truncate-lines-2"
+                href={result?.links?.members.replace("/api", "")}
+              >
+                {result.metadata.title}
+              </a>
+            </Header>
+            {result.metadata.description && (
+              <p className="truncate-lines-2 text size small text-muted mt-5">
+                {result.metadata.description}
+              </p>
+            )}
+
+            {(communityType ||
+              result.metadata.website ||
+              result.metadata.organizations) && (
+              <div className="wrap mt-5 text size small text-muted">
+                {communityType && (
+                  <CommunityTypeLabel
+                    transparent={communityTypeLabelTransparent}
+                    type={communityType}
+                  />
+                )}
+
+                {result.metadata.website && (
+                  <div className="rel-mt-1">
+                    <Icon name="linkify" />
+                    <a
+                      href={result.metadata.website}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      className="text-muted"
+                    >
+                      {result.metadata.website}
+                    </a>
+                  </div>
+                )}
+
+                {result.metadata.organizations && (
+                  <div>
+                    <Icon name="building outline" />
+                    {result.metadata.organizations.map((org, index) => {
+                      const separator = (index > 0 && ", ") || "";
+
+                      return (
+                        <span className="text-muted" key={org.name}>
+                          {separator}
+                          {org.name}
+                          {org.id && (
+                            <a
+                              href={`https://ror.org/${org.id}`}
+                              aria-label={`${org.name}'s ROR ${i18next.t(
```

Comment:
@mirekys The issue is, the Communities settings form (that also allows you to fill community metadata), contains remote select field that calls to /affiliations, which I imagine is some Invenio implementation that calls to ROR behind the scenes. Anyway, we will need to figure out how to handle this. Probably it will be necessary to override this input with our input and call to our affiliations and or ROR, but I find it problematic to generalize the UI, without knowing how it is going to work in reality and what I am going to be getting back. In my opinion, we will have to sit down on this for the next round.

Reply:
Ok let's leave it as is for now & deal with it when it becomes an actual problem